rambo/options.py

```diff
@@ -4,6 +4,19 @@
 from rambo.settings import SETTINGS, PROJECT_LOCATION, PROJECT_NAME
 from rambo.utils import get_env_var, set_env_var
 
+def address_option(address=None):
+    '''Validate address. If "dhcp", get an address in 172.28 network. If not supplied, set to default 192.168.33.10. Set as env var.
+
+    Args:
+        address (str): IP address to set.
+
+    Return address (str)
+    '''
+    if not address:
+        address = '192.168.33.10'
```

Comment:
We might as well take this opportunity to change this to the ip address [vagrantup.com](https://www.vagrantup.com/docs/networking/private_network.html) uses: 192.168.50.4